### PR TITLE
Telemetry: Privacy Policy

### PR DESCRIPTION
**Pull request by omefire**

### CODE REVIEW

**Comment by omefire** [comment]
@axemclion, @nikhilkh  to review

File: www/privacy/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page 
+title: Privacy Policy 
+---
+
+# Privacy Policy
+
+When you opt into telemetry, information about your usage of Apache Cordova is collected. The collected information consists of the following:
```
Comment (nikhilkh) [inline www/privacy/index.md]: Apache Cordova CLI

File: www/privacy/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+* IP address
+* OS type and version
```
Comment (nikhilkh) [inline www/privacy/index.md]: Node version?

**Comment by omefire** [comment]
http://imgur.com/FSDAifF

**Comment by omefire** [comment]
What it looks like: http://imgur.com/FSDAifF

File: www/privacy/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Cordova version
+* Commands executed, time of execution and status (success/failure)
+
+This information is saved in Google Analytics and handled by Google as described in their [privacy policy](http://www.google.com/policies/privacy/). See the documentation for instructions on how to disable telemetry if you prefer not to share your data.
+
+We use the gathered information to help us make our tool more useful and to better understand how it is used. We do not track or collect personally identifiable information or associate gathered data with any personally identifying information from other sources.
+
+By opting into telemetry, you consent to the collection of this data in the manner and for the purpose described above.
+
+Apache Cordova welcomes your questions or comments regarding this Privacy Policy. Send them to dev@cordova.apache.org
```
Comment (axemclion) [inline www/privacy/index.md]: The Apache Cordova __community__ welcomes your questions or comments regarding this Privacy Policy. Send them to dev@cordova.apache.org

File: www/privacy/index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+When you opt into telemetry, information about your usage of Apache Cordova CLI is collected. The collected information consists of the following:
+
+* IP address
+* OS type and version
+* Node version
+* Cordova version
+* Commands executed, time of execution and status (success/failure)
+
+This information is saved in Google Analytics and handled by Google as described in their [privacy policy](http://www.google.com/policies/privacy/). See the documentation for instructions on how to disable telemetry if you prefer not to share your data.
```
Comment (axemclion) [inline www/privacy/index.md]: Link to documentation that says how to disable telemetry.

File: www/privacy/index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+layout: page 
+title: Privacy Policy 
+---
+
+# Privacy Policy
```
Comment (axemclion) [inline www/privacy/index.md]: This is just the privacy policy for hte CLI. I think we should have that as a heading.

Comment (nikhilkh) [inline www/privacy/index.md]: +1 - only covers the CLI - not the website itself.

File: www/privacy/index.md (hunk 4 above)
Comment (nikhilkh) [inline www/privacy/index.md]: We should probably link to http://cordova.apache.org/docs/en/latest/reference/cordova-cli/index.html and add the docs for it.

Comment (omefire) [inline www/privacy/index.md]: +1

**Comment by omefire** [comment]
Thanks, comments addressed.

**Pull request closed without merging**